src/Core/Components/DataGrid/Columns/ColumnResizeOptions.razor

```diff
@@ -0,0 +1,45 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@typeparam TGridItem
+
+<FluentStack Class="resize-options" VerticalAlignment="VerticalAlignment.Center" HorizontalGap="4">
+    @if (ResizeType == DataGridResizeType.Discrete)
+    {
+        <FluentLabel>@Label</FluentLabel>
+
+        <FluentSpacer />
+        <FluentButton OnClick="HandleShrinkAsync">
```

Comment:
Please, for clean code, could you use the `@` sign to inform the user that this is a .NET method? `@HandleShrinkAsync`